packages/esm-active-visits-app/src/visits-summary/visit-detail-overview.scss

```diff
@@ -0,0 +1,228 @@
+@import "~@openmrs/esm-styleguide/src/vars";
+@import "~carbon-components/src/globals/scss/vars";
+@import "~carbon-components/src/globals/scss/mixins";
+@import "../root.scss";
+
+// Typography
+
+.caption01 {
```

Comment:
There are a lot of things in there which should be in root.scss and which may be even removed. Do we even need these (e.g., `caption01`, `bodyShort02`)?

Reply:
I agree, shifting it all to `root.scss`